supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/labels/Label.scala

```diff
@@ -0,0 +1,59 @@
+package cromwell.backend.impl.jes.labels
+
+/**
+  * Construct via Label.safeLabel
```

Comment:
The comment isn't wrong but I think there may have been a level of indirection introduced since it was written that makes it a bit confusing